### DISCUSSION TITLE
Help with making 1 year old project work again

### DISCUSSION THREAD

**Discussion by hkirsman** [vercel/next.js#18097] [Help] [answered]
Hey! I have this noobie situation where I created next.js site, put it live and it's running but now months later I tried to start it in local but I have error:

	+ export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin
	+ PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin
	+ npm run dev

	> sick-fits-frontend@1.0.0 dev /app
	> node server.js

	warn  - You are using a non-standard "NODE_ENV" value in your environment. This creates inconsistencies in the project and is strongly advised against. Read more: https://err.sh/next.js/non-standard-node-env
	Loaded env from /app/.env
	The static directory has been deprecated in favor of the public directory. https://err.sh/vercel/next.js/static-dir-deprecated
	info  - Using external babel configuration from /app/package.json
	error - ./pages/_app.js:54:6
	Syntax error: Support for the experimental syntax 'jsx' isn't currently enabled (54:7):

	  52 | 
	  53 |     return (
	> 54 |       <Container>
	     |       ^
	  55 |         <Page>
	  56 |           <Component {...pageProps} />
	  57 |         </Page>

	Add @babel/plugin-transform-react-jsx (https://git.io/vb4yd) to the 'plugins' section of your Babel config to enable transformation.

I could upload the project to some github repository if it would be helpful. It consits of backend service that also runs locally.

Thank you!

**Comment by hkirsman** [comment]
Now that I read it says I don't have jsx syntax enabled somewhere? How was that changed?

**Comment by jamesmosier** [comment, marked as answer]
Hi @hkirsman. It looks like you have a custom babel config:

> Using external babel configuration from /app/package.json

There is likely an issue or conflict in there. Could you share what your babel config is?

[From the docs](https://nextjs.org/docs/advanced-features/customizing-babel-config), you'll want to ensure you have `"presets": ["next/babel"]` as well.

**Reply by hkirsman** [reply]
package.json
```

{
  "name": "sick-fits-frontend",
  "version": "1.0.0",
  "description": "Sick Fits! An example React, GraphQL, Next and Apollo",
  "main": "index.js",
  "scripts": {
    "dev": "node server.js",
    "build": "next build",
    "start": "NODE_ENV=production node server.js",
    "test": "NODE_ENV=test jest --watch",
    "test-win": "SET NODE_ENV=test&& jest --watch",
    "heroku-postbuild": "next build"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@babel/core": "^7.12.0",
    "babel-plugin-styled-components": "^1.11.1",
    "date-fns": "^2.16.1",
    "downshift": "^3.4.8",
    "enzyme": "^3.11.0",
    "enzyme-adapter-react-16": "^1.15.5",
    "express": "^4.17.1",
    "isomorphic-unfetch": "^3.1.0",
    "lodash.debounce": "^4.0.8",
    "merge-stream": "^2.0.0",
    "next": "^9.5.5",
    "next-routes": "^1.4.2",
    "nookies": "^2.4.0",
    "nprogress": "^0.2.0",
    "prop-types": "^15.7.2",
    "react": "^16.14.0",
    "react-adopt": "^0.6.0",
    "react-dom": "^16.14.0",
    "react-facebook": "^8.1.4",
    "react-ga": "^2.7.0",
    "react-load-script": "0.0.6",
    "react-retina-image": "^2.0.5",
    "react-stripe-checkout": "^2.6.3",
    "react-transition-group": "^4.4.1",
    "vinyl-buffer": "^1.0.1",
    "waait": "^1.0.5"
  },
  "devDependencies": {
    "babel-jest": "^24.9.0",
    "babel-plugin-module-resolver": "^3.2.0",
    "browser-sync": "^2.26.7",
    "casual": "^1.6.2",
    "enzyme-to-json": "^3.4.0",
    "gulp": "^3.9.1",
    "gulp-bump": "^3.1.3",
    "gulp-clean": "^0.4.0",
    "gulp-concat": "^2.6.1",
    "gulp-cssmin": "^0.2.0",
    "gulp-filter": "^6.0.0",
    "gulp-git": "^2.9.0",
    "gulp-if": "^3.0.0",
    "gulp-imagemin": "^6.1.0",
    "gulp-notify": "^3.2.0",
    "gulp-rename": "^1.4.0",
    "gulp-sass": "^4.0.2",
    "gulp-sass-glob": "^1.1.0",
    "gulp-shell": "^0.7.1",
    "gulp-tag-version": "^1.3.1",
    "gulp-uglify": "^3.0.2",
    "gulp-watch": "^5.0.1",
    "gulp.spritesmith": "^6.10.1",
    "jest": "^24.9.0"
  },
  "jest": {
    "setupTestFrameworkScriptFile": "<rootDir>/jest.setup.js",
    "testPathIgnorePatterns": [
      "<rootDir>/.next/",
      "<rootDir>/node_modules/"
    ],
    "transform": {
      "\\.(gql|graphql)$": "jest-transform-graphql",
      ".*": "babel-jest",
      "^.+\\.js?$": "babel-jest"
    }
  },
  "//": "This is our babel config, I prefer this over a .babelrc file",
  "babel": {
    "env": {
      "development": {
        "presets": [
          "next/babel"
        ],
        "plugins": [
          [
            "styled-components",
            {
              "ssr": true,
              "displayName": true
            }
          ]
        ]
      },
      "production": {
        "presets": [
          "next/babel"
        ],
        "plugins": [
          [
            "styled-components",
            {
              "ssr": true,
              "displayName": true
            }
          ]
        ]
      },
      "test": {
        "presets": [
          [
            "next/babel",
            {
              "preset-env": {
                "modules": "commonjs"
              }
            }
          ]
        ],
        "plugins": [
          [
            "styled-components",
            {
              "ssr": true,
              "displayName": true
            }
          ]
        ]
      }
    }
  }
}

```

Should there be babel* something file somewhere? Don't see in project root .

**Reply by hkirsman** [reply]
I've put the frontend on Github https://github.com/hkirsman/tsitaatcom_site_frontend

**Reply by hkirsman** [reply]
It was my fault - I was overwriting env varible with this:

    NODE_ENV=dev

**Reply by hkirsman** [reply]
Tx for helping out!
